examples/example-shader/src/lib.rs

```diff
@@ -133,23 +131,16 @@ fn sky(dir: Vec3, sun_position: Vec3) -> Vec3 {
     );
     let mut l0 = 0.1 * fex;
     l0 += sun_e * 19000.0 * fex * sundisk;
-    let mut tex_color = lin + l0;
-    tex_color *= Vec3::splat(0.04);
-    tex_color += Vec3::new(0.0, 0.001, 0.0025) * 0.3;
+    let tex_color = lin + l0;
 
     // Tonemapping
-    let white_scale = 1.0 / uncharted2_tonemap(TONEMAP_WEIGHTING);
-    let curr = uncharted2_tonemap(((2.0 / LUMINANCE.pow(4.0)).log2()) * tex_color);
-    let color = curr * white_scale;
-
-    color.pow(1.0 / (1.2 + (1.2 * sunfade)))
+    let color = tex_color.min(Vec3::splat(1024.0));
```

Comment:
I'm not sure about most of what's going on here, for example, I'm not even sure why the sky model chose to embed the tone-mapping operator in here to begin with (or even why it's doing srgb conversions tbh). It might be nicer to hoist all of that out of `sky` all together? 

One thing I do notice is that your change seems to remove the application of `sunfade`. I suspect this was done to make the sun seem brighter.

Reply:
> I'm not even sure why the sky model chose to embed the tone-mapping operator in here to begin with
> ...
> It might be nicer to hoist all of that out of sky all together?

Yeah it would... the tonemapping is really just because the sky is being displayed directly... so it makes sense to put it in the main frag shader and not in `sky()`.

and while it does remove sunfade from here, I suspect that the application of `sunfade` here was really just a hacked in "auto brightness slider" (which really shouldnt be done as a gamma anyway) based on a vaguely related value... it's still being used in the calculation of the rayleigh coefficient right after it gets defined.